### PR TITLE
Add config and rake task for harvesting outside records

### LINKED ISSUE #874: Create a system for automated re-indexing / harvesting of external GeoBlacklight documents.
See: sul-dlss/earthworks#718
https://github.com/sul-dlss/earthworks/blob/master/lib/tasks/earthworks.rake#L110

### PR DESCRIPTION
**Pull request by eliotjordan**
Closes #874

### CODE REVIEW

File: config/environments/staging.rb [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# frozen_string_literal: true
+
+Rails.application.configure do
```
Comment (tpendragon) [inline config/environments/staging.rb:3]: Suggestion: There's a lot of settings here that we probably want to keep the same between staging/prod. Can we just include the prod file like we do in https://github.com/pulibrary/figgy/blob/db_lando/config/environments/staging.rb?

File: config/schedule.rb [hunk 2]
```diff
@@ -11,13 +11,42 @@
...
 # Run a weekly rake task to harvest thumbnail in incomplete states.
 # These could be from new records or from records
 # where there was a previous error during harvesting.
-every :saturday, at: "11:00 AM", roles: [:db] do
+every :saturday, at: "11:00 AM", roles: [:index] do
   rake "gblsci:images:harvest_retry"
 end
+
+# Harvest new records from other GeoBlacklight sites
+every "0 1 1 * *", roles: [:index] do
```
Comment (tpendragon) [inline config/schedule.rb:26]: Suggestion: Can you add some comments about what "0 1 1 * *" etc means? Maybe convert this to a more wordy format that whenever supports?

Comment (eliotjordan) [reply config/schedule.rb:26]: Whenever doesn't support english translations for these, but I can add some comments.

**Comment by eliotjordan** [comment]
@tpendragon Added comments to the harvesting cron schedules and dried up the staging config.

**Pull request merged**